wagtail/admin/views/editing_sessions.py

```diff
@@ -52,6 +52,7 @@ def ping(request, app_label, model_name, object_id, session_id):
         )
 
     session.last_seen_at = timezone.now()
+    session.is_editing = request.GET.get("editing", False)
```

Comment:
I forgot to raise this in the previous PR, but now that we're also taking additional data into account, are we sure we want to use `GET` for this endpoint? Especially considering that we also do database writes, which isn't exactly best practice for `GET` requests as they are meant to be safe and idempotent.

Reply:
Yep, I'd be on board with making it a POST. (It occurred to me while writing the unit test for the "session ID not found" case that repeatedly loading that URL would endlessly create new session records, but I didn't make the sensible logical leap from there...)